.changelog/11573.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+connect: Support Vault auth methods for the Connect CA Vault provider.
```

Comment:
small nit -- can we call out which Vault auth methods are now supported?

IIUC, today, we would support all thru this PR. 

Thinking that as Vault may or may not add more methods, it'd be useful to look thru release notes to trace features additions, deprecations, etc